hibernate-core/src/main/java/org/hibernate/hql/spi/id/inline/IdsClauseBuilder.java

```diff
@@ -0,0 +1,30 @@
+package org.hibernate.hql.spi.id.inline;
+
+import java.util.List;
+
+/**
+ * Builds the where clause that wraps the identifiers to be updated/deleted.
+ *
+ * @author Vlad Mihalcea
+ */
+public abstract class IdsClauseBuilder {
+
+	private final List<Object[]> ids;
+	private final String[] columns;
+
+	protected IdsClauseBuilder(
+			String[] columns, List<Object[]> ids) {
+		this.columns = columns;
+		this.ids = ids;
+	}
+
+	public String[] columns() {
+		return columns;
+	}
+
+	public List<Object[]> ids() {
```

Comment:
Changes applied. I noticed that the naming convention was conflicting with the `CteValuesListBuilder`.